ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-release-4.14.yaml

```diff
@@ -142,6 +142,22 @@ tests:
     test:
     - ref: windows-e2e-operator-test-byoh
     workflow: windows-e2e-operator-test-platform-none-vsphere
+- as: vsphere-proxy-e2e-operator
```

Comment:
I don't think you need to add this explicitly to `release-4.14`. The bot should automatically sync this.